### ISSUE TITLE
how could I implement user theme?

### ISSUE THREAD

**Issue by kufii** [benevbright/react-navigation-collapsible#71]
I have a theme object with a bunch of colors passed through the screenProps of my navigator.

I'm having a trouble thinking of a way of changing the collapsibleBackgroundStyle based on this? I can access the screenProps from the collapsibleComponent itself but how would I access it within withCollapsible

so for example

```javascript
withCollapsible(Screen, {
	collapsibleComponent: ExtendedHeader,
	collapsibleBackgroundStyle: {
		height: 60,
		backgroundColor: '#fff' // I'd like to use the user theme here
	}
});
```

**Comment by quangvo09** [comment]
+1

**Comment by benevbright** [comment]
Planned to support this with v5

**Comment by benevbright** [comment]
resolved by #103
